src/quacc/calculators/mrcc/io.py

```diff
@@ -72,17 +72,23 @@ def write_mrcc(file_path: Path | str, atoms: Atoms, parameters: MRCCParamsInfo)
         - mult : int <-- Multiplicity of the system.
     """
 
+    write_geom = True
+
     with Path.open(file_path, "w") as file_path:
         # Write the MRCC input file
         for key, value in parameters["mrccinput"].items():
             file_path.write(f"{key}={value}\n")
 
         # Write the MRCC blocks
-        file_path.write(f"{parameters['mrccblocks']} \n")
+        if parameters["mrccblocks"] is not None:
+            file_path.write(f"{parameters['mrccblocks']} \n")
+
+        if "geom" in parameters["mrccinput"]:
+            write_geom = False
 
-        # If the geometry is not provided in the MRCC blocks, write it here.
-        ghost_list = []  # List of indices of the ghost atoms.
-        if "geom" not in parameters["mrccblocks"]:
+        if write_geom:
```

Comment:
I am a bit confused. Is it ever even possible in this logic for `write_geom` to be `False`?

Reply:
This has been included specifically for SKZCAM functions. We generate the geometry using mrccblocks so the geometry is actually in mrccblocks since the ASE atoms objects cannot account for the additional embedding environment we need to create.

Reply:
Ah sorry, I had misread it!